src/main/java/org/thymeleaf/util/NumberUtils.java

```diff
@@ -285,6 +285,71 @@ private static DecimalFormatSymbols computeDecimalFormatSymbols(
     
     
     
+    public static String formatCurrency(final Number target, final Locale locale) {
+
+        Validate.notNull(locale, "Locale cannot be null");
+
+        if (target == null) {
+            return null;
+        }
+
+        NumberFormat format = null;
+        
+        if (target instanceof Double || target instanceof Float || target instanceof BigDecimal ||
```

Comment:
I just copied from the other methods. I guess it's unnecessary because it has to be one of the specified types.